index.js

```diff
@@ -182,7 +182,14 @@ class RNCallKeep {
   };
 
   sendDTMF = (uuid, key) => RNCallKeepModule.sendDTMF(uuid, key);
-
+  /**
+   * @description when Phone call is active, Android control the audio service via connection service. so this function help to toggle the audio to Speaker or wired/ear-piece or vice-versa 
+   * @param {*} uuid 
+   * @param {*} routeSpeaker
+   * @returns Audio route state of audio service
+   */
```

Comment:
Documentation for the method also needs to be in `README.md`

Reply:
updated here https://github.com/react-native-webrtc/react-native-callkeep/pull/295/commits/b0a2fdab532ad246516816d4acdcbe58d20333b2

please review